scripts/release/prepare-packages.js

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env node
+
+/**
+ * @license Copyright (c) 2003-2023, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/* eslint-env node */
+
+'use strict';
+
+const readline = require( 'readline' );
+const releaseTools = require( '@ckeditor/ckeditor5-dev-release-tools' );
+
+const abortController = new AbortController();
+
+// Windows does not understand CTRL+C attached to `process`.
+if ( process.platform === 'win32' ) {
+	const rl = readline.createInterface( {
+		input: process.stdin,
+		output: process.stdout
+	} );
+
+	rl.on( 'SIGINT', () => process.emit( 'SIGINT' ) );
+}
```

Comment:
This works weird on Windows:
- I have the impression that I have to press `CTRL+C` several times and wait a few seconds for the script to be terminated.
- Additionally, if there is no `release` directory, the scripts hangs:
   <img src="https://user-images.githubusercontent.com/56868128/236206040-f94d7788-4dc2-4ce3-9e78-611e7846a134.png" width="500px">


The good news is we don't need this whole `if` at all 😄 

Question: whether we have the `readline.createInterface()` or not, terminated script throws this error. Is it expected?

![obraz](https://user-images.githubusercontent.com/56868128/236205278-4ea63371-9818-4975-a729-287e2dca9fd4.png)

Reply:
The unhandled rejection error should be fixed in https://github.com/ckeditor/ckeditor5-dev/pull/869/commits/828a7e214b9688f62650ea739b0b84c4099f4c25.

Reply:
Thanks, it works fine now.